assets/src/components/selection/event-select/index.js

```diff
@@ -42,27 +43,35 @@ export const EventSelect = ( {
 	selectedEventId,
 	isLoading,
 } ) => {
-	if ( isLoading || isEmpty( events ) ) {
-		return <Placeholder key="placeholder"
-			icon="calendar"
-			label={ __( 'EventSelect', 'event_espresso' ) }
-		>
-			{ isLoading ?
-				<Spinner /> :
-				__(
-					'There are no events to select from. You need to create an event first.',
-					'event_espresso',
-				)
-			}
-		</Placeholder>;
+	if ( isEmpty( events ) ) {
+		return (
+			<Fragment>
+				<Placeholder
+					icon="calendar"
+					label={ __( 'EventSelect', 'event_espresso' ) }
+				>
+					{ ! isLoading && isEmpty( events ) ?
```

Comment:
isn't this second call to `isEmpty()` redundant because we are already within the brackets for the first `isEmpty()` check?
ie: if `events` is NOT empty... then how do we get to this code in the first place?

Reply:
ya it's redundant.  This went through a variety of conditional checks so I probably missed this when I switched the parent condition.  Good catch.